src/features/voters/components/VotersList.tsx

```diff
@@ -13,11 +14,16 @@ export const VotersList = (): JSX.Element => {
   }
   return (
     <div className="space-y-1">
-      {(
-        data ??
-        Array(5)
-          .fill(0)
-          .map((_, i) => ({ recipient: i }))
+      {(data
+        ? data.reduce((unique: Attestation[], item: Attestation) => {
+            if (!unique.find((obj) => obj.recipient === item.recipient)) {
+              unique.push(item);
+            }
+            return unique;
+          }, [])
+        : Array(5)
+            .fill(0)
+            .map((_, i) => ({ recipient: i }))
```

Comment:
Why do you need this empty array?

Reply:
that's not written by me, I just kept the code here, but I should change it to just do the things five times

Reply:
I think you can just add `[]` as a fallback, I don't think we need these recipient indices.

Reply:
I agree, just added it